.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,94 @@
+terraform {
+  required_version = ">=0.13"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+provider "aws" {
+  region = "us-east-1"
+}
```

Comment:
Just to comment something we mentioned on the call: it would be cool to tag resources created by this with a creation timestamp and perhaps something else to identify them as being created for automated-testing. This will help us identify them and "garbage collect" them if the "Terraform Destroy" step fails for some reason.

According to https://engineering.deptagency.com/best-practices-for-terraform-aws-tags you can add default tags in this section and all resources created will have those tags.